### PR TITLE
Yc/restructure upd tend

### PR DESCRIPTION
**Pull request by yairchn**
I am moving the clipping of diagnostic variables to the filter of the prognostic variables (this requires defining a small non zero value). The hope is that this reduce the number of diagonals in the Jacobian matrix

### CODE REVIEW

File: src/TurbulenceConvection/types.jl [hunk 1]
```diff
@@ -457,6 +457,7 @@ struct EDMFModel{
 }
     surface_area::FT
     max_area::FT
+    small_area::FT
```
Comment (charleskawczynski) [inline src/TurbulenceConvection/types.jl:460]: How does this differ from `minimum_area`?

Comment (yairchn) [reply src/TurbulenceConvection/types.jl]: only if we have `>=` in the conditions. I changed it to `>` and now we can use only `a_min`

**Review by charleskawczynski — APPROVED** [review]

**Comment by yairchn** [comment]
Bors r+

**Comment by yairchn** [comment]
Bors r+

**Pull request merged**